examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1733,7 +1772,7 @@ class GLTFMaterialsPbrSpecularGlossinessExtension {
 		material.aoMapIntensity = 1.0;
 
 		material.emissive = materialParams.emissive;
-		material.emissiveIntensity = 1.0;
+		material.emissiveIntensity = materialParams.emissiveIntensity === undefined ? 1.0 : materialParams.emissiveIntensity;
```

Comment:
I'm surprised the extension is permitted with spec/gloss materials — do you know if this was an exception to the rule for other new PBR extensions?

Reply:
I was reading the spec because I also wanted to double-check that, and it's not explicitly excluded.

Seems based on these discussions here (where you had chimed in) the exclusion was pulled out:
- https://github.com/KhronosGroup/glTF/pull/1994#issuecomment-886786649

Reply:
Ok, no concerns then!